src/app/app.component.ts

```diff
@@ -1,10 +1,12 @@
 import { Component } from '@angular/core';
+import { ROUTER_DIRECTIVES } from '@angular/router';
 
 @Component({
     moduleId: module.id,
     selector: 'app-root',
     templateUrl: 'app.component.html',
-    styleUrls: ['app.component.css']
+    styleUrls: ['app.component.css'],
+    directives:[ROUTER_DIRECTIVES]
```

Comment:
you can remove the directive ROUTER_DIRECTIVES would be imported through the RouterModule